spec/immutable-data-structures.html

```diff
@@ -49,8 +49,10 @@ <h1>Record.fromEntries ( _iterable_ )</h1>
           1. Perform ? RequireObjectCoercible(_iterable_).
           1. Let _fields_ be a new empty List.
           1. Let _adder_ be a new Abstract Closure with parameters (_key_, _value_) that captures _fields_ and performs the following steps when called:
+            1. If Type(_key_) is Symbol, throw a *TypeError* exception.
```

Comment:
we could drop this as the `toString` AO we are using does this.

Also `Object.fromEntries` does https://tc39.es/ecma262/#sec-topropertykey calling `toPrimitive`